### PR TITLE
Upgrade Django to 2.2.15

### LINKED ISSUE #118: Upgrade Django to 2.2.15
See:

- https://www.djangoproject.com/weblog/2020/aug/03/django-bugfix-releases-309-2215/
- https://github.com/azavea/docker-django/actions/runs/186054019

### PR DESCRIPTION
**Pull request by coalkettler**
Upgrade Django to the most recent patch release.

See: https://www.djangoproject.com/weblog/2020/aug/03/django-bugfix-releases-309-2215/

---

**Testing**

See: https://github.com/azavea/docker-django/actions/runs/193687339

Fixes #118

### CODE REVIEW

File: 2.2/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 gunicorn==20.0.4
-django==2.2.14
+django==2.2.15
 psycopg2==2.8.5
 gevent==1.4.0
```
Comment (rbreslow) [inline 2.2/requirements.txt:4]: Looks like `1.5.0` is out.

**Review by rbreslow — APPROVED** [review]
👍

File: 2.2/requirements.txt (hunk 1 above)
Comment (coalkettler) [reply 2.2/requirements.txt:4]: And several other versions! They switched up the versioning scheme so the most recent version is `20.6.2`. Unfortunately everything `1.5.0` and up fails to build on Alpine, details in #111. I tried to build `20.6.2.` as well on the off chance that they changed something but I ran into the same failure.

I don't want us to fall too far behind on `gevent` so I'm going to add it to the next sprint and we can decide if we want to take care of it.

Comment (rbreslow) [reply 2.2/requirements.txt:4]: Oh, yeah, I forgot. I neglected to mention the 20.x series as that'd be a major version bump.

**Pull request merged**